config.example/group_vars/all.yml

```diff
@@ -220,6 +220,17 @@ maas_single_node_install: true
 
 maas_kvm: false
 
+################################################################################
+# SOFTWARE MODULES (SM)                                                        #
+################################################################################
```

Comment:
Not sure if this stuff will conflict with any of the recent slurm changes.

Reply:
These were all moved to the slurm-cluster playbook